spec/models/mixins/relationship_mixin_spec.rb

```diff
@@ -216,11 +216,20 @@ def self.name
           has_ancestry
           self.table_name = "services"
           include RelationshipMixin
+          self.default_relationship_type = "ems_metadata"
```

Comment:
I was also under the impression that we didn't want this line at all: the reason I (for the moment) need it in https://github.com/ManageIQ/manageiq/pull/21226 was because it's covering the bug that this PR is fixing.

Reply:
I put this in because this is the default relationship. If you note it is different from the relationship that we are testing.